File: Triangle.java
public class Triangle extends Figure {
    private double side1, side2, side3;

    public Triangle(String color, double side1, double side2, double side3) {
        super(color);
        this.side1 = side1;
        this.side2 = side2;
        this.side3 = side3;
    }

    @Override
    public double perimetro() {
        return side1 + side2 + side3;
    }

    @Override
    public double area() {
        double s = perimetro() / 2;
        return Math.sqrt(s * (s - side1) * (s - side2) * (s - side3));
    }
}

File: Triangle.py
import math

class Triangle(Figure):
    def __init__(self, color, side1, side2, side3):
        super().__init__(color)
        self.side1 = side1
        self.side2 = side2
        self.side3 = side3

    def perimetro(self):
        return self.side1 + self.side2 + self.side3

    def area(self):
        s = self.perimetro() / 2
        return math.sqrt(s * (s - self.side1) * (s - self.side2) * (s - self.side3))
